nnpackage/spec/10_packaging_and_manifest.md

```diff
@@ -108,6 +108,7 @@ Configuration file is a human readable plain text file having one `key=value` in
 For example
 ```
 BACKENDS=cpu
+# leading/trailing space is ignored
```

Comment:
@glistening , comment is added but if you think this is redundant, I'll remove this